wallet/wallet.h

```diff
@@ -1052,4 +1067,16 @@ struct amount_msat wallet_total_forward_fees(struct wallet *w);
  */
 const struct forwarding *wallet_forwarded_payments_get(struct wallet *w,
 						       const tal_t *ctx);
+
+/**
+ * Load remote_ann_node_sig and remote_ann_bitcoin_sig
+ *
+ * @w: wallet containing the channel
+ * @id: channel database id
+ * @remote_ann_node_sig: location to load remote_ann_node_sig to
+ * @remote_ann_bitcoin_sig: location to load remote_ann_bitcoin_sig to
```

Comment:
Like mentioned above, with this signature I would have expected the function to allocate the signatures on the heap. But in that case we're missing a `ctx` to allocate from.

Reply:
Thank you! Now I add it.